### DISCUSSION TITLE
Can I change the Transmit Frequency on the CubeSatSim?

### DISCUSSION THREAD

**Discussion by big-roboto** [alanbjohnston/CubeSatSim#318] [General]
Hello all,

The satellite transmits on 434.9, but could I get some guidance on how we can change the Tx frequency? I was thinking of changing the frequency for maybe a few kHz. 

Thanks,
Ron

**Comment by alanbjohnston** [comment]
Hi Ron,

Yes, you can within the range of 420 - 450 MHz.

If you are running the beta v1.3 software for the blue PCBs, then you log into your Pi Zero and type this command:

`CubeSatSim/config -F`

and you will be prompted for the transmit and receive frequencies, then it will reboot using the new frequencies. 

If you are running the v1.2 software for the white PCBs, then you need to do a search and replace inside the rpitx.py file where you will replace 434.9 with another frequency in MHz.  You will need to reboot then for the changes to take effect.

If you aren't sure which version, try the beta approach, but the `CubeSatSim/config -F` command won't work which tells you that you are running the v1.2 software instead.

Good luck!
Alan

**Comment by alanbjohnston** [comment]
What frequency do you want?On Jun 7, 2024, at 13:16, Jaehyun Park ***@***.***> wrote:﻿
Hello, I have a follow up question on this. Can I configure the specific frequency that I want?
